packages/opentelemetry-resources/src/IResource.ts

```diff
@@ -51,4 +51,9 @@ export interface IResource {
    * @returns the newly merged Resource.
    */
   merge(other: IResource | null): IResource;
+
+  /**
+   * Returns the schema URL of the resource.
+   */
+  getSchemaUrl(): string; // Add this line
```

Comment:
This is a breaking change - `@opentelemetry/resources` is a stable SDK package, and new properties added to interfaces must be optional.